server/mentions.go

```diff
@@ -284,7 +286,7 @@ func (p *NotificationsParser) sendUserActivity(userActivity *UserActivity) error
 	urlParams := url.Values{}
 	urlParams.Set("context", string(jsonContext))
 
-	appID, err := p.pluginStore.GetAppID(userActivity.TenantID)
+	appID, err := p.pluginStore.GetAppID(p.tenantID)
```

Comment:
Couldn't we just use: 
```suggestion
	appID, err := p.pluginStore.GetAppID(p.msteamsAppClient.GetTenantID())
```

Reply:
I just thought it looked cleaner with the shorter access.

Reply:
Fixed